<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWF</class>
 <widget class="QDialog" name="CWF">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Completed Work Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="2">
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Lab</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_lab"/>
     </item>
     <item row="1" column="1" rowspan="2">
      <widget class="QComboBox" name="comboBox_Machine">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Machine Model</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Completed Work Description</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Request Duo </string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>Accident</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_2">
           <property name="text">
            <string>Safty Concern</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Cost</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Labor</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Material</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_labor"/>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_Material"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Email</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>447</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CWF</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>468</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>452</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CWF</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>473</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
